devtools/environment.yml

```diff
@@ -35,6 +35,9 @@ dependencies:
   - jupyterlab~=3.2
   - nbdime~=3.1.0
 
+  # Helpful for profiling PUDL
+  - py-spy
```

Comment:
Does `py-spy` benefit from being installed via `conda`? Or should it be added to the `dev` extras?

Reply:
I suppose it doesn't need to live in the environment.yml file. What do you mean by dev extras?

Reply:
The "extras" are additional dependencies that aren't needed in all cases, but which are specified with a label in the `setup.py` file and can be optionally installed by users that are going to use that optional aspect of the package. Right now we've got `dev`, `test`, and `doc` extras. Right now most of the stuff in the `environment.yml` file is either packages that depend on C libraries, or stuff that we only use in our interactive analysis (Jupyter). It could really go in either place and probably be fine, since we're only using it interactively at this point and I think we're all using the conda environments.

Reply:
Got it. I think it makes sense to live in the `environment.yml` given we're using it interactively and we're using the conda envs.